contracts/strategies/auraBAL/StrategyBase.sol

```diff
@@ -0,0 +1,112 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.9;
+
+import "../../../interfaces/IBasicRewards.sol";
+import "../../../interfaces/balancer/IBalancer.sol";
+import "../../../interfaces/balancer/IAsset.sol";
+import "../../../interfaces/balancer/IBalPtDeposit.sol";
+
+contract AuraBalStrategyBase {
```

Comment:
HandlerBase.sol doesn't seem to inherit from `IRewardHandler.sol`, so neither do `AuraHandler.sol` and `BBUSDHandler.sol`. I guess it compiles fine because you're hard casting it to an interface anyway in `Strategy.sol`: `IRewardHandler(_tokenHandler).sell();`. But then again there's no compile time checking to make sure this won't fail.

Reply:
done